### DISCUSSION TITLE
New version of fagci, different graphics

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#370] [General]
It seems that there is a new version of fagci, it has different graphics, perhaps partially more intuitive 
https://www.youtube.com/watch?v=gZYY5v09Bo0

**Comment by OE1MWW** [comment]
with this version you need a magnifying glass or have to be an eagle to read the numbers on the screen

**Reply by gartnl** [reply]
And it seems rather more complicated to use from the preview. I like the simple bandscope it was.
